contracts/tec/package.json

```diff
@@ -0,0 +1,90 @@
+        "compile": "sol-compiler",
+        "watch": "sol-compiler -w",
+        "clean": "shx rm -rf lib generated-artifacts generated-wrappers",
+        "generate_contract_wrappers": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output generated-wrappers --backend ethers",
+        "lint": "tslint --format stylish --project . --exclude ./generated-wrappers/**/* --exclude ./generated-artifacts/**/* --exclude **/lib/**/* && yarn lint-contracts",
+        "coverage:report:text": "istanbul report text",
+        "coverage:report:html": "istanbul report html && open coverage/index.html",
+        "profiler:report:html": "istanbul report html && open coverage/index.html",
+        "coverage:report:lcov": "istanbul report lcov",
+        "test:circleci": "yarn test",
+        "contracts:gen": "contracts-gen",
+        "lint-contracts": "solhint -c ../.solhint.json contracts/**/**/**/**/*.sol"
+    },
+    "config": {
+        "abis": "./generated-artifacts/@(IExchange|TEC|TestLibs|TestMixins).json",
+        "abis:comment": "This list is auto-generated by contracts-gen. Don't edit manually."
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/0xProject/0x-monorepo.git"
+    },
+    "license": "Apache-2.0",
+    "bugs": {
+        "url": "https://github.com/0xProject/0x-monorepo/issues"
+    },
+    "homepage": "https://github.com/0xProject/0x-monorepo/contracts/extensions/README.md",
+    "devDependencies": {
+        "@0x/abi-gen": "^2.0.3",
+        "@0x/contracts-gen": "^1.0.2",
+        "@0x/contract-wrappers": "^7.1.0",
+        "@0x/contracts-test-utils": "^3.0.4",
+        "@0x/dev-utils": "^2.1.0",
+        "@0x/sol-compiler": "^3.0.3",
+        "@0x/tslint-config": "^3.0.0",
+        "@types/lodash": "4.14.104",
+        "@types/node": "*",
+        "chai": "^4.0.1",
+        "chai-as-promised": "^7.1.0",
+        "chai-bignumber": "^3.0.0",
+        "dirty-chai": "^2.0.1",
+        "make-promises-safe": "^1.1.0",
+        "mocha": "^4.1.0",
+        "npm-run-all": "^4.1.2",
+        "shx": "^0.2.2",
+        "solhint": "^1.4.1",
+        "tslint": "5.11.0",
+        "typescript": "3.0.1"
+    },
+    "dependencies": {
+        "@0x/base-contract": "^4.0.3",
+        "@0x/contracts-asset-proxy": "1.0.2",
+        "@0x/contracts-exchange-libs": "^1.0.5",
+        "@0x/contracts-exchange": "1.0.2",
+        "@0x/contracts-erc20": "1.0.2",
+        "@0x/contracts-erc721": "1.0.2",
+        "@0x/contracts-utils": "^2.0.4",
+        "@0x/order-utils": "^6.1.0",
+        "@0x/types": "^2.0.2",
+        "@0x/typescript-typings": "^4.0.0",
+        "@0x/utils": "^4.1.0",
+        "@0x/web3-wrapper": "^5.0.0",
+        "ethereumjs-util": "^5.1.1",
+        "ethereum-types": "^2.0.0",
+        "lodash": "^4.17.11"
```

Comment:
Does this package really need all these deps?

Reply:
I'll double check, might not need a few.